ansible_collections/arista/avd/plugins/README.md

```diff
@@ -206,6 +210,35 @@ bgp_peer_groups:
       password: "{{ bgp_vault_password | arista.avd.encrypt(passwd_type='bgp', key='IPv4-UNDERLAY-PEERS') }}"
 ```
 
+#### OSPF passwords
+
+OSPF passwords are encrypted/decrypted based on the interface name (e.g., Ethernet1) and for message-digest-key the authentication algotithm (in the list [md5, sha1, sha256, sha384, sha512]) and the key ID (between 1 and 255).
```

Comment:
I think we are mixing auth algorithm (simple or md5) with hash algorithm in the text here.
I would suggest to create two separate password types. One for ospf simple and one for ospf message digest. The hash algorithm can then be passed as option to the filter.